keps/sig-instrumentation/1668-trace-popagating/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: trace information popagation
+kep-number: 1668
+authors:
+ - "@hase1128"
+ - "@KobayashiD27"
+ - "@fenggw-fnst"
+ - "@zhijianli88"
+ - "@Hellcatlk"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-api-machinery
+status: provisional
+creation-date: 2020-09-01
+last-updated: 2021-01-21
+reviewers:
+ - "@dashpole"
+ - "@serathius"
+approvers:
+ - "@dashpole"
+prr-approvers:
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.21"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.21"
+  beta: "v1.22"
+  stable: "v1.25"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+   - name: TracePopagating
```

Comment:
```suggestion
   - name: PropagateContextTrace
```

Reply:
Just to double confirm, do you want to say PropagateTraceContext instead of  PropagateContextTrace, since in this KEP we have never said "ContextTrace" so far.

Reply:
ContextTrace also fine